modules/rosa-imds-machine-pools.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: CONCEPT
+[id="rosa-imds-machine-pools_{context}"]
+= Instance Metadata Service on machine pools
+
+Instance metadata allows you to configure and manage a running instance. Instance metadate also allows you to access user data that you need to launch your instance.
```

Comment:
Honestly, I am not sure what practical information a reader can get from these two sentences. I suggest to either skip it or provide a link to AWS documentation (preferred) or rephrase the way that  someone without context knowledge can understand what the metadata or IMD is and what is it useful for.